packages/plaid-funding-source/pages/index.tsx

```diff
@@ -32,6 +32,26 @@ const Home: NextPage = () => {
      */
     const [linkToken, setLinkToken] = useState<string | null>(null);
 
+    /**
+     * The body text that is returned from the /on-demand-authorizations endpoint
+     */
+    const [bodyText, setBodyText] = useState<string | null>(null);
+
+    /**
+     * The button text that is returned from the /on-demand-authorizations endpoint
+     */
+    const [buttonText, setButtonText] = useState<string | null>(null);
+
+    /**
+     * The ODA link that is returned from the /on-demand-authorizations endpoint
+     */
+    const [odaLink, setOdaLink, odaLinkRef] = useState<string | null>(null);
```

Comment:
Depending on the direction that is gone with [this comment](https://github.com/Dwolla/integration-examples/pull/14/files#r1052671880), if `odaLink` remains unused, it can be removed.

Reply:
It will remain unused. Will remove.

Reply:
`odaLink` will end up being used after all. It gets used in a check to see if it's value exist now.